ui/apps/platform/src/Containers/Dashboard/DashboardPage.tsx

```diff
@@ -28,17 +28,31 @@ function DashboardPage() {
     const hasReadAccessForNode = hasReadAccess('Node');
     const hasReadAccessForSecret = hasReadAccess('Secret');
 
+    const hasReadAccessForSummaryCounts =
+        hasReadAccessForAlert ||
+        hasReadAccessForCluster ||
+        hasReadAccessForDeployment ||
+        hasReadAccessForImage ||
+        hasReadAccessForNode ||
+        hasReadAccessForSecret;
+
     return (
         <>
-            <PageSection variant="light" padding={{ default: 'noPadding' }}>
-                {hasReadAccessForAlert &&
-                    hasReadAccessForCluster &&
-                    hasReadAccessForDeployment &&
-                    hasReadAccessForImage &&
-                    hasReadAccessForNode &&
-                    hasReadAccessForSecret && <SummaryCounts />}
-            </PageSection>
-            <Divider component="div" />
+            {hasReadAccessForSummaryCounts && (
+                <>
+                    <PageSection variant="light" padding={{ default: 'noPadding' }}>
+                        <SummaryCounts
+                            Cluster={hasReadAccessForCluster}
+                            Node={hasReadAccessForNode}
+                            Violation={hasReadAccessForAlert}
+                            Deployment={hasReadAccessForDeployment}
+                            Image={hasReadAccessForImage}
+                            Secret={hasReadAccessForSecret}
```

Comment:
Curious why you chose TitleCase prop names?

Reply:
Also, 6 values as props is a shortcut that I might change to one prop that has an object value, which is more forward thinking, in case some other prop is ever needed.